satpy/readers/viirs_l2.py

```diff
@@ -0,0 +1,155 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
```

Comment:
Those lines are redundant and do not add anything useful.

Reply:
Removed in c6c97a440